docs/administration/user-permissions.md

```diff
@@ -0,0 +1,99 @@
+
+- for reviewers add the role **CiviRemote User**.
+- for receiving contacts add the roles **CiviRemote User** and **CiviRemote Funding**.
+
+!!! question
+    It can be handy to allow the creation of Drupal user accounts for unauthenticated users, for example new applicants. This is done at *Arbeit und Leben*, I think. How is this done exactly? I want to add this as a tip.
+
+!!! note
+    The CiviRemote User Roles are automatically synchronised with the Drupal user account during login. If there are problems with the synchronisation, open the drupal user overview at `admin/people`. Select the user that you want to assign the roles to and perform the action **CiviRemote: Match contact(s)** and afterward **CiviRemote: Synchronise CiviRemote roles**.
+
+
+## Funding Program
+
+After creating a funding program, it will be listed at **Funding > Funding Programs**. There are several actions available, namely **Edit Recipients**, **Edit permissions** and **Edit initial funding case permissions**. These three settings have to be configured correctly before opening the program for applicants.
+
+### Edit Recipients
+
+When an applicant creates an application, one field of the application form asks who is going to be the recipient of the funding. The recipient is not necessarily identical to the applicant. The field in the application form is a dropdown menu where the correct recipient needs to be chosen. The values that are shown in the dropdown menu need to be configured at **Edit Recipients**.
+
+As an example, consider a funding program that funds organizations. The application is not created by the organization but by an employee of that organization. In this case, the applicant is the employee and the recipient is the organization. If both exist as contacts in CiviCRM and are connected by a relationship of type _Employee_, the configuration would look as follows:
+
+![](../img/permissions_edit_recipients.png)
+
+The dropdown menu will then show all organizations that have an _Employee_ relationship with the applicant that is logged in and creating the application.
+
+### Edit permissions
+
+These permissions are related to a funding program, independently of a specific funding case or application.
+
+The following permissions exist:
+
+* **Application: create** allows contacts to create a new application for this funding program.
+* **Application: apply** allows contacts to create a new application and apply it in one step.
+* **View** specifies which funding programs are available to create an application for. It also specifies which funding programs are visible in the CiviCRM funding program list.
+
+The permissions **Application: create** and **Application: apply** are usually granted to applicants. Both implicitly include the permission **View**. The **View** permission is typically given to reviewers and site admins.
```

Comment:
Going through your proposals one by one and seeing, that we both had the same thought of removing the sentence :ok_hand: 

I added the following paragraph (guessing that "it" refers to the View permission)

> Some Drupal roles automatically imply funding program permissions. Users with the Drupal permission **CiviCRM: administer Funding Program Manager** implicitly have the **View** permission. Users with the Drupal permission **CiviCRM: administer CiviCRM** implicitly have every permission.

Reply:
Actually I'd not call it Drupal permission. In case CiviCRM does not run in Drupal, but e.g. WordPress, it's definitely not a Drupal permission :-) I think the sentence with "roles" is not necessary then.

Reply:
I would like to keep the first sentence ("Somoe Drupal roles ...") and just replace "Drupal permission" by "permission" in the following sentences. Two reasons: first is, that the funding framework is explicitly referring to Drupal. There is even a heading in this page that is called "Drupal permission". Second, the topic about permissions is for me the most difficult / confusing so far. Mostly because there are many different pages where permissions need to be set. This is why I want to refer to Drupal in some way - it would help me as a reader to get aware that this is not about funding permissions anymore.